src/envoy/http/authn/authn_utils.cc

```diff
@@ -24,45 +24,68 @@ namespace AuthN {
 namespace {
 class JsonIterCallback : public Logger::Loggable<Logger::Id::filter> {
  public:
-  // The json iterator callback function
-  static bool JsonIteratorCallback(const std::string&,
-                                   const Envoy::Json::Object&,
-                                   istio::authn::JwtPayload* payload);
+  // The json iterator callback function for extracting Jwt claims
+  static bool JsonIteratorCallbackForJwtClaims(
+      const std::string&, const Envoy::Json::Object&,
+      istio::authn::JwtPayload* payload);
+  // The json iterator callback function for exracting Jwt audience
+  static bool JsonIteratorCallbackForJwtAudience(
```

Comment:
Doesn't need this function. In fact, even the other can be lambda now, given it's so much simpler.

Reply:
Done.